raven_line_count.sh

```diff
@@ -0,0 +1 @@
+cat pg17192.txt | wc -l
```

Comment:
this is totally legit, and it works.  So there's no problem, just an fyi the give and take of using scripts.  If the script included a loop or something more complex it still makes sense, however, when the script is a one liner, it may also just go in the makefile.  The give and take is complexity in the makefile versus more scripts floating around in the root directory.